src/queries/user.ts

```diff
@@ -0,0 +1,20 @@
+import { User } from '../entity/user';
+import { CustomError } from '../error-handling';
+
+export async function user(_: unknown, args, context): Promise<User> {
```

Comment:
Alguns pontos sobre essa função:

- Eu costumo recomendar nomer funções com verbos, porque geralmente deixa mais claro o que elas fazem. Algo como `getUser`

- Imagino q vc tenha feito isso pra poder passar direto a referência da função lá no reoslver. Porém, eu acho que valeria mais a pena vc limitar essa função pra receber apenas o que ela precisa. Assim não fica uma função solta recebendo parâmetros inúteis. O que acha?

- Quando vc cria a função solta assim, o Typescript passa a não saber de qual tipo são os argumentos, e assim vc fica com a tipagem desabilitada e perde os benefícios do Typescript. Acho que seria uma boa vc tipar o args e o context.

Reply:
tipagem foi refatorada com interfaces